bentoml/_internal/configuration/containers.py

```diff
@@ -59,30 +69,47 @@ def _is_ip_address(addr: str) -> bool:
         return False
 
 
+RUNNER_CFG_SCHEMA = {
+    "batching": {
+        # for review: should we allow configuring batching per-method as well as per-runner?
```

Comment:
ideally it should allow per-method batching options but practically very few use cases will need that. this is something we can add in the future right?

Reply:
Yes, that should be backwards-compatible.

Reply:
Going to delete this line; we can do this later.